extensions/blocks/conversation/components/participants-controls.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { DropdownMenu, TextControl, Button } from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+function AddParticipantButton( { className, onAdd, participants = [] } ) {
+	return (
+		<div className={ `${ className }__participant` }>
+			<Button
+				className={ `${ className }__add-button` }
+				label={ __( 'Add Participant', 'jetpack' ) }
+				onClick={ () => onAdd( `Participant ${ participants.length + 1 }` ) }
```

Comment:
Non-blocking: Should this be translated?

Reply:
yes 👍

Reply:
let me kick it for a next PR